volume/drivers/fake/fake_test.go

```diff
@@ -184,23 +186,24 @@ func testInitForCloudBackups(t *testing.T, d *driver) (string, *api.CloudBackupC
 		CredentialUUID: credid,
 	}
 
-	id, err := d.cloudBackupCreate(req)
+	taskId, id, err := d.cloudBackupCreate(req)
```

Comment:
This is calling the internal method which seems wrong. Should use the interface?

Reply:
This is fake driver setting things up.